dds/DCPS/InstanceState.cpp

```diff
@@ -5,6 +5,8 @@
  * See: http://www.opendds.org/license.html
  */
 
+#include <sstream>
```

Comment:
Safety profile excludes the C++ standard library streams.  See SafetyProfileStreams.h for helpful utility functions.